src/state_machine.rs

```diff
@@ -407,6 +413,8 @@ impl StateMachine {
             events: Vec::new(),
         };
 
+        info!("{} - Resume", machine.current());
```

Comment:
shall this info more about the `state before resume` ?

Reply:
It's just so we can see in the logs when resume happened. I didn't need it to be more detailed. When we actually need that, we can always add it.